exiftool.go

```diff
@@ -4,21 +4,24 @@ import (
 	"bufio"
 	"bytes"
 	"encoding/json"
+	"errors"
 	"fmt"
 	"io"
 	"os"
 	"os/exec"
 	"sync"
-
-	"errors"
+	"time"
 )
 
 var executeArg = "-execute"
-var initArgs = []string{"-stay_open", "True", "-@", "-", "-common_args"}
+var initArgs = []string{"-stay_open", "True", "-@", "-"}
 var extractArgs = []string{"-j"}
 var closeArgs = []string{"-stay_open", "False", executeArg}
 var readyTokenLen = len(readyToken)
 
+// WaitTimeout specifies the duration to wait for exiftool to exit when closing before timing out
+var WaitTimeout = time.Second
```

Comment:
I think that `WaitTimeout`should be unexported.
You perhaps let it exported so that users can specify another timeout duration, that's an interesting feature, but the custom duration should be specified using a dedicated function. This way, we can change implementation whenever we want.

Reply:
Do you want to allow clients to change the wait timeout after initializing `Exiftool`?

> This way, we can change implementation whenever we want.

Nit: any exported field, method, or function creates an interface which limits any implementation if you want to maintain backwards compatibility. e.g. the exported option functions are part of an interface provided by the `exiftool` package

The main difference I see between directly setting the config in the `Exiftool` struct and the option functions is the ability to change config for an instance of `Exiftool`.

Reply:
I'm sorry, I didn't read well, I thought that `Timeout` was an Exiftool struct attribute but it is not. Forget what I told :)

Reply:
Haha, we could make it an `Exiftool` struct attribute if you want the timeout to be configurable per instance of the struct.